app/views/dataset/local-spending.html

```diff
@@ -0,0 +1,40 @@
+{% extends "layout.html" %}
+
+
+{% block page_title %}
+  Dataset - data.gov.uk
+{% endblock %}
+
+{% block content %}
+
+<main id="content" class="local-authority-choice" role="main">
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p class="heading-first">Local Authorities datasets collection</p>
+      <h1 class="heading-large">Spending over £500</h1>
+      <p class="lede">Select Local Authorities</p>
+    </div>
+
+      <form id="select-LA" action="" method="GET" class="column-two-thirds">
+        <fieldset>
+          {% for organisation in data.local_organisations() %}
```

Comment:
Sorry to be a pain, rather than ```data.local_organisations()``` it would probably be better to use ```data.organisations```.  Other than that, looks great.